tests/unit_tests/udf_wrapper_params/token_classification/make_data_row_functions.py

```diff
@@ -0,0 +1,95 @@
+# base params, used for input/output rows if no other params are given
+device_id=None
+bucketfs_conn="bfs_conn"
+sub_dir="sub_dir"
+model_name="model"
+
+text_data="text"
+text_docid = 1
+text_start = 0
+text_end = 6
+
+agg_strategy_simple="simple"
+
+token="token"
+token_start = 2
+token_end = 4
+entity_type="ENTITY_TYPE"
+score=0.1
+error_msg = None
+
+def make_input_row(device_id=device_id, bucketfs_conn=bucketfs_conn, sub_dir=sub_dir,
+                   model_name=model_name, text_data=text_data, aggregation_strategy=agg_strategy_simple):
+    """
+    creates an input row for token classification without span usage as a list,
+    using base params for all params that are not specified.
+    """
+    return [(device_id, bucketfs_conn, sub_dir, model_name,text_data, aggregation_strategy)]
+
+def make_output_row(bucketfs_conn=bucketfs_conn, sub_dir=sub_dir,
+                    model_name=model_name, text_data=text_data, aggregation_strategy=agg_strategy_simple,
+                    start=token_start, end=token_end, word=token,
+                    entity=entity_type, score=score, error_msg=error_msg):
+    """
+    creates an output row for token classification without span usage as a list,
+    using base params for all params that are not specified.
+    the found token is called "word" in our non span udf output,
+    while the type/class of the found token is called "entity".
+    """
+    return [(bucketfs_conn, sub_dir, model_name,text_data, aggregation_strategy,
+             start,end,word, entity,score, error_msg)]
+
+def make_model_output_for_one_input_row(number_entities:int, aggregation_strategy=agg_strategy_simple,
+                                        entity_group=entity_type, score=score, word=token, start=token_start, end=token_end):
+    """
+    makes the output the model returns to the udf for one input row.
+    the found token is called "word" in the model output,
+    while the type/class of the found token is called "entity_group".
+    unless aggregation_strategy == "none", then the type/class of the found
+    token is called "entity" in the model output.
+    returns a list of number_entities times the model output row.
```

Comment:
Use proper docstring formatting for return values, see other projects. I think, bucketfs has it.

Reply:
also added to new ticket